qadence/backends/braket/backend.py

```diff
@@ -173,12 +173,12 @@ def expectation(
         param_values: dict[str, Tensor] = {},
         state: Tensor | None = None,
         measurement: Measurements | None = None,
-        error: Errors | None = None,
+        error: Noise | None = None,
         endianness: Endianness = Endianness.BIG,
     ) -> Tensor:
         if error is not None:
             logger.warning(
-                f"Errors of type {error} are not implemented for expectation yet. "
+                f"Noise of type {error} are not implemented for expectation yet. "
```

Comment:
```suggestion
                f"Error of type {error} are not implemented for expectation yet. "
```

Reply:
I made it to be both Noise here, still, can turn it to "Error of type {noise} ..."